### PR TITLE
Add advection-dominated stabilization examples

### LINKED ISSUE #10027: Add examples for advection dominated regimes in navier stokes module
## Rationale
Illustrate necessity of stabilization for high Reynolds number

## Description
Add examples for both SUPG stabilized Q2Q1 and SUPG,PSPG stabilized Q1Q1

## Impact
Exemplifies how INS can now be used in much larger range of Reynolds numbers

### PR DESCRIPTION
**Pull request by lindsayad**
Includes:
  - SUPG stabilization with Q2Q1
  - SUPG,PSPG stabilization with Q1Q1

Closes #10027

### CODE REVIEW

File: modules/navier_stokes/src/kernels/INSBase.C [hunk 1]
```diff
@@ -276,8 +276,12 @@ INSBase::tau()
...
   RealVectorValue U(_u_vel[_qp], _v_vel[_qp], _w_vel[_qp]);
   Real h = _current_elem->hmax();
-  return _alpha / std::sqrt((2. * U.norm() / h) * (2. * U.norm() / h) +
-                            9. * (4. * nu / (h * h)) * (4. * nu / (h * h)));
+  if (!_transient_term)
+    return _alpha / std::sqrt((2. * U.norm() / h) * (2. * U.norm() / h) +
+                              9. * (4. * nu / (h * h)) * (4. * nu / (h * h)));
+  else
+    return _alpha / std::sqrt((1. / _dt) * (1. / _dt) + (2. * U.norm() / h) * (2. * U.norm() / h) +
+                              9. * (4. * nu / (h * h)) * (4. * nu / (h * h)));
```
Comment (jwpeterson) [inline modules/navier_stokes/src/kernels/INSBase.C]: This (and especially in the Jacobian) is a lot of duplicated code just to turn off the one contribution...
```
Real transient_part = _transient_term ? 4. / (_dt * _dt) : 0.;
return _alpha / std::sqrt(transient_part + ...);
```
seems cleaner.

Comment (jwpeterson) [reply modules/navier_stokes/src/kernels/INSBase.C]: The above also switches to the more common 2/dt form.

File: modules/navier_stokes/test/tests/ins/mms/supg/supg_pspg_adv_dominated_mms.i [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+[]
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  xmin = 0
+  xmax = 1.0
+  ymin = 0
+  ymax = 1.0
+  elem_type = QUAD9
```
Comment (jwpeterson) [inline modules/navier_stokes/test/tests/ins/mms/supg/supg_pspg_adv_dominated_mms.i]: This problem should use QUAD4s, right?. Besides this and the `pspg=true` parameter, are there any other differences between the two input files? Seems like they should just be combined into one and the tests controlled with CLI args.

**Review by jwpeterson — CHANGES_REQUESTED** [review]

**Comment by lindsayad** [comment]
@jwpeterson Changing to 2/dt required regolding of the transient mms tests and adding the 1/rho to PSPG stabilization required regolding of all other mms tests involving PSPG. I'm going to re-run our paper's convergence studies and make sure our results are still the same.

**Comment by jwpeterson** [comment]
> @jwpeterson Changing to 2/dt required regolding of the transient mms tests and adding the 1/rho to PSPG stabilization required regolding of all other mms tests involving PSPG. I'm going to re-run our paper's convergence studies and make sure our results are still the same.

OK, you can save those changes for another PR then if you want?

**Review by jwpeterson — APPROVED** [review]

**Pull request merged**